pulp_container/app/settings.py

```diff
@@ -45,6 +45,10 @@
     "application/vnd.wasm.config.v1+json": [
         "application/vnd.wasm.content.layer.v1+wasm",
     ],
+    # delta manifest
+    "application/vnd.redhat.delta.config.v1+json": [
```

Comment:
Specs say that manifest should be of special delta type as well, this would be great to clarify with the authors, and if confirmed then this would require us to add it to the DB manifests media type choices, and update the json schema validator